addons/website_sale/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,84 @@
+---Fields in module 'website_sale'---
+website_sale / payment.transaction      / sale_order_id (many2one)      : module is now 'website_quote' ('website_sale')
+# Nothing to do
+
+website_sale / product.public.category  / website_meta_description (text): NEW
+# Nothing to do
+
+website_sale / product.public.category  / website_meta_keywords (char)  : NEW
+# Nothing to do
+
+website_sale / product.public.category  / website_meta_title (char)     : NEW
+# Nothing to do
+
+website_sale / product.template         / rating_ids (one2many)         : NEW relation: rating.rating
+# Nothing to do
+
+website_sale / res.country.group        / website_pricelist_ids (many2many): NEW relation: website_pricelist
+# Nothing to do
+
+website_sale / res.partner              / last_website_so_id (many2one) : NEW relation: sale.order
+# Nothing to do
```

Comment:
We can try to deduce this looking for a sales order that comes from website from this partner.